lib/main_common.pm

```diff
@@ -635,6 +636,33 @@ sub load_x11regression_documentation {
     loadtest "x11regressions/libreoffice/libreoffice_double_click_file";
 }
 
+sub load_x11regression_other {
+    if (check_var("DESKTOP", "gnome")) {
+        loadtest "x11regressions/brasero/brasero_launch";
+        loadtest "x11/gnomeapps/gnome_documents";
+        loadtest "x11regressions/totem/totem_launch";
+    }
+    if (check_var('DISTRI', 'sle') && sle_version_at_least('12-SP2')) {
+        loadtest "x11regressions/shotwell/shotwell_import";
+        loadtest "x11regressions/shotwell/shotwell_edit";
+        loadtest "x11regressions/shotwell/shotwell_export";
+        loadtest "virtualization/yast_virtualization";
+        loadtest "virtualization/virtman_view";
```

Comment:
Are they only available for SLE? I guess they could run on Tumbleweed as well.
You can have an investigation on it for next PR? :)

Reply:
In terms of shotwell, we plan use gnome-photos to replace it in SLE15. And then shotwell cases will be replaced by gnome-photos cases. So, didn't port it to TW. 
Since the yast2 virtualization interface are different in SLE and TW. We will consider port virtualization cases to TW in the future. :)

Reply:
I got it, thanks.